### ISSUE TITLE
Feedback on /userland

### ISSUE THREAD

**Issue by groundwater** [electron/electron.atom.io#609]
First off, I love it ❤️ 💯 🎉 🎈 

1. I think it's so good, the format almost doesn't do it justice. I would suggest making the links on the front page into *buttons* or *panels*, something that really draws my attention to entice me to click them. Something like the following: ![image](https://cloud.githubusercontent.com/assets/538488/20906890/439fcb04-baff-11e6-98ce-703608aa4292.png)
2. Can you put counts on the packages listed on the other page? I recognize they'd be out of date from the moment of inception, but how about some indicator of how popular/active?

**Comment by simurai** [comment]
> Can you put counts on the packages listed on the other page? I recognize they'd be out of date from the moment of inception, but how about some indicator of how popular/active?

2. Maybe instead of showing numbers it could be simplified with just a range of 5... like for the stars it would just show from ⭐️  to ⭐️⭐️⭐️⭐️⭐️  stars maybe same with ⬇️  downloads. Then the absolute number matters less and it can be updated once a week/month.

3. The [wall of contributors avatars](http://electron.atom.io/userland/github_contributors) is pretty neat. But for me the avatrs in [userland/package_authors](http://electron.atom.io/userland/package_authors) don't show up. The `url` seems fine, but they show as having a`1x1` size. Maybe a security issue that doesn't load them properly?

4. The link + description is a bit hard to parse. Like it's hard to see where the description starts. Maybe use the list like http://electron.atom.io/community or a different link/text color?

**Comment by zeke** [comment]
Opened #617, which addresses some of this stuff.

> The url seems fine, but they show as having a1x1 size. Maybe a security issue that doesn't load them properly?

Which browser are you using, @simurai? I'm not able to reproduce in Chrome, Safari, or Firefox on macOS.

**Comment by simurai** [comment]
Chrome. Yep, other browsers are fine. Also Chrome in incognito mode is fine. Probably just some local glitch. Interestingly I also don't see your avatar on https://www.npmjs.com/~zeke.

**Comment by MarshallOfSound** [comment]
>  Interestingly I also don't see your avatar on https://www.npmjs.com/~zeke.

I don't either.  @zeke is secretly a 👻

**Comment by zeke** [comment]
Hmm.

Can you see the image when you visit the raw URL?
https://s.gravatar.com/avatar/8f344b1c4bdcfc28bd848e97e94c3523?size=496&default=retro

Can you see others avatars on npm?

https://www.npmjs.com/~electron
https://s.gravatar.com/avatar/341d73a1c4caa8fec88ed24181562113?size=496&default=retro

**Comment by zeke** [comment]
Maybe you two are using an ad blocker that blocks gravatar domains?

**Comment by MarshallOfSound** [comment]
Ad block is completely disabled and those image URL's are 503'ing for me.

> Error 503 Backend fetch failed

Probably an issue on Gravatar's side of things

**Comment by simurai** [comment]
> Maybe you two are using an ad blocker that blocks gravatar domains?

Ouch.. yes.. that's it in my case. Ok, whitelisted Gravatar.

**Comment by MarshallOfSound** [comment]
I wanted to be super sure so I ran `curl`

```
C:\Users\Samuel>curl https://s.gravatar.com/avatar/8f344b1c4bdcfc28bd848e97e94c3523?size=496&default=retro
<!DOCTYPE html>
<html>
  <head>
    <title>503 Backend fetch failed</title>
  </head>
  <body>
    <h1>Error 503 Backend fetch failed</h1>
    <p>Backend fetch failed</p>
    <h3>Guru Meditation:</h3>
    <p>XID: 1008010235</p>
    <hr>
    <p>Varnish cache server</p>
  </body>
</html>
```

99% sure that isn't my fault 😆

**Comment by zeke** [comment]
@MarshallOfSound do you get something different if you wrap that URL in quotes?

**Comment by zeke** [comment]
Hopefully we don't have to set up a gravatar proxy..

**Comment by zeke** [comment]
Closing this out. We have followup issues for stuff mentioned in this issue, e.g. #622
